app/javascript/mastodon/features/list_timeline/index.jsx

```diff
@@ -73,9 +73,9 @@ class ListTimeline extends PureComponent {
     this.disconnect = dispatch(connectListStream(id));
   }
 
-  UNSAFE_componentWillReceiveProps (nextProps) {
+  componentDidUpdate (prevProps) {
     const { dispatch } = this.props;
-    const { id } = nextProps.params;
+    const { id } = prevProps.params;
```

Comment:
That doesn't sound correct, the new `id` should be used in all actions below.

Reply:
Good catch, thanks!